packages/Pagination/src/Pagination.js

```diff
@@ -0,0 +1,106 @@
+import React from "react";
+import PropTypes from "prop-types";
+import ArrowItem from "./components/ArrowItem/ArrowItem";
+import CurrentPageItem from "./components/CurrentPageItem/CurrentPageItem";
+import ElipsisItem from "./components/EllipsisItem/ElipsisItem";
+import PageItem from "./components/PageItem/PageItem";
+import * as sc from "./Pagination.styles";
+
+const propTypes = {
+  /** The number of current active page */
+  currentPage: PropTypes.number.isRequired,
+
+  /** The number of other pages that will be visible around the current/active page (not hidden by elipsis). Can be set to small, medium, or large */
+  pagesOnEachSide: PropTypes.number,
+
+  /** Callback to be executed when current page is changed. */
+  onChange: PropTypes.func.isRequired,
+
+  /** The number of total pages. */
+  totalPages: PropTypes.number.isRequired,
+};
+
+const defaultProps = {
+  pagesOnEachSide: "medium",
```

Comment:
So should this be a string or number? In the propTypes definition above you're having `pagesOnEachSide: PropTypes.number,`